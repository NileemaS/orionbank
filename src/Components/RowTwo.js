import React from "react";
import { Image } from "react-bootstrap";
import Btnlink from "./Btnnavlink";
import '../App.css';

export default function RowTwo() {
  return (
<>

<div className="container text-center">    
  <h3>What We Can Do Today</h3>
  <br/>
  <div className="row">
    <div className="col-sm-3">
      <Image src="/img/imgbitcoin.jpg" className="Image-responsive" style={{width:"70%"}} alt="Image" />
      <h4>We accept <br/> Bitcoins <br/> With Other Currencies</h4>
    </div>
    <div className="col-sm-3"> 
      <Image src="/img/imgbond.jpg" className="Image-responsive" 
      style={{width:"70%"}} alt="Image" />
      <h4>Invest in Our <br/> Bonds</h4>    
    </div>
    <div className="col-sm-3">
      <Btnlink varName = "Create Account" varHref = "/CreateAccount/" />
      <Btnlink varName = "Deposit" varHref = "/Deposit/" />
    </div>
    <div className="col-sm-3">
      <Btnlink varName = "Withdraw" varHref = "/Withdraw/" />
      <Btnlink varName = "All Data" varHref = "/AllData/" />
    </div>  
  </div>

</div>
</>
);}