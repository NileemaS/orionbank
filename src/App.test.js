import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json';
import RowTwo from "./Components/RowTwo";


test("RowTwo matches snapshot", ()=>{
  const wrapper = shallow(<RowTwo />);

  expect (toJSON (wrapper)).toMatchSnapshot();

})