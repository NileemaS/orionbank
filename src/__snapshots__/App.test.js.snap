// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowTwo matches snapshot 1`] = `
<Fragment>
  <div
    className="container text-center"
  >
    <h3>
      What We Can Do Today
    </h3>
    <br />
    <div
      className="row"
    >
      <div
        className="col-sm-3"
      >
        <Image
          alt="Image"
          className="Image-responsive"
          fluid={false}
          rounded={false}
          roundedCircle={false}
          src="/img/imgbitcoin.jpg"
          style={
            Object {
              "width": "70%",
            }
          }
          thumbnail={false}
        />
        <h4>
          We accept 
          <br />
           Bitcoins 
          <br />
           With Other Currencies
        </h4>
      </div>
      <div
        className="col-sm-3"
      >
        <Image
          alt="Image"
          className="Image-responsive"
          fluid={false}
          rounded={false}
          roundedCircle={false}
          src="/img/imgbond.jpg"
          style={
            Object {
              "width": "70%",
            }
          }
          thumbnail={false}
        />
        <h4>
          Invest in Our 
          <br />
           Bonds
        </h4>
      </div>
      <div
        className="col-sm-3"
      >
        <Btnlink
          varHref="/CreateAccount/"
          varName="Create Account"
        />
        <Btnlink
          varHref="/Deposit/"
          varName="Deposit"
        />
      </div>
      <div
        className="col-sm-3"
      >
        <Btnlink
          varHref="/Withdraw/"
          varName="Withdraw"
        />
        <Btnlink
          varHref="/AllData/"
          varName="All Data"
        />
      </div>
    </div>
  </div>
</Fragment>
`;
